Pod/Classes/BONChain.h

```diff
@@ -78,6 +79,11 @@ typedef BONChain *BONCNonnull (^BONTagComplexStyles)(BONGeneric(NSArray, BONTag
 
 @property (copy, nonatomic, readonly) BONChainBaselineOffset baselineOffset;
 
+/**
+ *  Hyphenation is attempted when the ratio of the text width (as broken without hyphenation) to the width of the line fragment is less than the hyphenation factor. When the paragraph’s hyphenation factor is 0.0, the layout manager’s hyphenation factor is used instead. When both are 0.0, hyphenation is disabled.
```

Comment:
As far a I understand, it's basically a percentage. 1 means it will 100% use hyphens and 0 means it will never use them.